app/models/practice.rb

```diff
@@ -89,7 +89,11 @@ def has_facility?
 
   # views
   def views
-    Ahoy::Event.where_props(practice_id: id).count
+    Ahoy::Event.where(name: 'Practice show').where_props(practice_id: id).where("properties->>'is_duplicate' is null").count
```

Comment:
⚠️ just explicitly calling out it is the "Practice show" event here as well as filtering for `is_duplicate` `nil`